packages/morph/src/morph.js

```diff
@@ -319,6 +319,8 @@ function addNodeTo(node, parent) {
 
         return clone
     }
+
+    return parent;
```

Comment:
I'm not sure it should return parent sincr that is not an added node and it would make the debugger print thr wrong statement. It should probably return null or another falsy value and the caller should handle it properly.

Reply:
Yes, agreed, good catch. I've returned `null` from the function instead of `parent` and made the following changes to [line 198](https://github.com/alpinejs/alpine/pull/2528/files/02653339f78115c2a5f731c476e2e110d6c6c998#diff-16dbe79737c2ee37a02e810f6102eac94023aa7204051573a1463b3a846f5824R198) :

```diff
- let added = addNodeTo(currentTo, from)
+ let added = addNodeTo(currentTo, from) || {}
```

The variable `added` is only used for the debugger; and I'm assuming having a message of `Add element: undefined` is tolerable.